test/matlab_experimental/ctTestSetUp.m

```diff
@@ -0,0 +1,20 @@
+clear all
+
+% Copy library to test folder
+ctTestPath;
+
+% Load Cantera
+rootDir = getenv('CANTERA_ROOT');
+ctName = '/test/matlab_experimental/libcantera_shared.so';
```

Comment:
This appears specific to a Linux environment. Is it possible to implement this in a way that this can be run from any platform? (same in `ctTestTearDown.m`)

You do have some switches implemented in `ctTestPath`, which would also apply here:

```Matlab
if ispc
    [...]
elseif ismac
    [...]
elseif isunix
    [...]
end
```

Reply:
`/test/matlab_experimental` is not installation-specific. `ctTestSetUp` always temporarily copies the Cantera shared library from the build directory into the test directory and deletes it after `ctTestTearDown`. This is how the legacy Matlab test suite sets up the test. I don't think this intermediate step is necessary.

Reply:
But isn't it true that `libcantera_shared.so` refers to a Linux platform? Presumably, some users may want to run the test suite on Windows?